src/schema/section/submission.js

```diff
@@ -22,7 +22,6 @@ export const submission = (data = {}) => {
     }),
     Attachments: form.general('submission.attachments', {
       Method: form.text(attachmentType)
-    }),
-    Locked: data.Locked
```

Comment:
I couldn't find anywhere this was being used so I just removed it. @suzubara @christopherhuii is that OK?